hack/update-import-aliases.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+GOIMPORTS_PKG="golang.org/x/tools/cmd/goimports"
+GOIMPORTS_VERSION="v0.1.5"
+
+cd "${SCRIPT_ROOT}"
+ROOT_PATH=$(pwd)
+
+source "hack/util.sh"
+util::install_tools ${GOIMPORTS_PKG} ${GOIMPORTS_VERSION}
+
+IMPORT_ALIASES_PATH="${ROOT_PATH}/hack/.import-aliases"
+INCLUDE_PATH="(${ROOT_PATH}/cmd|\
+${ROOT_PATH}/pkg/apiserver|${ROOT_PATH}/pkg/kubeapiserver|\
+${ROOT_PATH}/pkg/storage|${ROOT_PATH}/pkg/synchromanager|\
+${ROOT_PATH}/pkg/utils|${ROOT_PATH}/pkg/version)"
+
+ret=0
+# We can't directly install preferredimports by `go install` due to the go.mod issue:
+# go install k8s.io/kubernetes/cmd/preferredimports@v1.21.3: k8s.io/kubernetes@v1.21.3
+#   The go.mod file for the module providing named packages contains one or
+#   more replace directives. It must not contain directives that would cause
+#   it to be interpreted differently than if it were the main module.
```

Comment:
I do not use preferredimports binary but just lift the file to `hack/tools/preferredimports/preferredimports.go` 
 AS `The go.mod file for the module providing named packages contains one or
more replace directives. It must not contain directives that would cause
 it to be interpreted differently than if it were the main module`
like `karmada` does

Reply:
OK，you suggestion may be better.  @Iceber  help to review the updated